src/openrct2-ui/windows/Ride.cpp

```diff
@@ -49,6 +49,7 @@
 #include <openrct2/object/ObjectRepository.h>
 #include <openrct2/object/StationObject.h>
 #include <openrct2/rct1/RCT1.h>
+#include <openrct2/rct12/RCT12.h>
```

Comment:
Is this header really required? After the import we should use the internal structures.

Reply:
I see, not ideal but will do for now.

Reply:
There is a good reason this function resides in `GetStationStyleFromIdentifier()` - it should **only** be used for export. It only supports entrances present in vanilla.

Reply:
Changed to alphabetic sorting, analogous to the ride type and vehicle type dropdown.